c++/src/kj/compat/gzip.h

```diff
@@ -29,12 +29,14 @@ KJ_BEGIN_HEADER
 
 namespace kj {
 
+#define KJ_GZ_BUF_SIZE 4096
```

Comment:
Prefer constants over macros:

```suggestion
constexpr size_t KJ_GZ_BUF_SIZE = 4096;
```

And can this be moved inline the `_` namespace? I don't think this constant is intended to be public.